packages/glimmer-runtime/lib/builder.ts

```diff
@@ -279,6 +279,14 @@ export class ElementStack {
     return fragmentBounds;
   }
 
+  insertTextBefore(nextSibling: Node, text: string): Fragment {
```

Comment:
I'm a bit sad that we now need so much machinery around `{{foo}}` when 90%+ of the time it's going to be just simple strings, but I don't have any ideas to improve it at the moment :/